### DISCUSSION TITLE
Problems about avatar and theme mode

### DISCUSSION THREAD

**Discussion by Eternal579** [cotes2020/jekyll-theme-chirpy#1191] [Q&A] [answered]
I cannot use the local picture to change my avatar, like this:
![5994af7eadf58140c8ecaa8b66c73e8](https://github.com/cotes2020/jekyll-theme-chirpy/assets/98520126/cc85222c-a745-48bc-a596-03f467d81b4b)
What's more, i did not change the code about theme mode, but i just can't use the button to switch between light and dark mode.
![9fced0e13df5373e7b1be87c2cc3fec](https://github.com/cotes2020/jekyll-theme-chirpy/assets/98520126/b2729e6f-4e59-4067-8a6c-5d4f2f717e26)

**Comment by abhishek7h** [comment, marked as answer]
Remove the CDN
